### ISSUE TITLE
Houses are locked

### ISSUE THREAD

**Issue by webbedspace** [softdorothy/glider_pro#4]
This doesn't really matter since one can still use Glider Edit to unlock them, but it seems a little against the spirit of open source that you can't initially open them in the game's own editor. :/

**Comment by drvink** [comment]
As you said, it's trivial to unlock them...I doubt keeping them locked was intentional, probably just an oversight.

John, thank you so much for releasing the Glider and Glider Pro source. This truly brings me back to my childhood.

**Comment by softdorothy** [comment]
Ha ha, I forgot that houses can be locked. Sadly, I don't have a machine setup where I can even run the House Editor to unlock them. Maybe some enterprising individual with an emulator will unlock them and commit them. It doesn't matter to me.

**Comment by amatecha** [comment]
Yeah, fire up Basilisk II or such, would be pretty trivial to do.

**Comment by Platskies** [comment]
Done, and made a pull request. I made separate unlocked archives to preserve which houses were and weren't unlocked.
